time_capture/time_capture/doctype/time_capture/time_capture.py

```diff
@@ -190,6 +194,44 @@ def send_weekly_time_capture_reminders():
 			message=message,
 			now=True,
 		)
+	frappe.db.set_value("Time Capture Settings", "last_notification_sent_on", today)
```

Comment:
```suggestion
	settings.db_set("last_notification_sent_on", today)
```

Weiter oben gibt es auch noch ein `frappe.db.get_single_value("Time Capture Settings", ...)`, das jetzt unnötig ist.